run

```diff
@@ -432,7 +432,15 @@ build_local() {
 
 copy_gds_files_() {
     local root_folder=${1:-${TOP}}
-    local cufile_search="${root_folder}/temp/cuda/include:${root_folder}/temp/cuda/lib64"  #"/usr/local/cuda/include:/usr/local/cuda/lib64 ${PREFIX:-}/include:${PREFIX:-}/lib ${CONDA_PREFIX:-}/include:${CONDA_PREFIX:-}/lib ${root_folder}/temp/cuda/include:${root_folder}/temp/cuda/lib64"
+    local cufile_search="${root_folder}/temp/cuda/include:${root_folder}/temp/cuda/lib64"
+    if [[ ! -z "${CONDA_PREFIX+x}" ]]
+    then
+      cufile_search="${CONDA_PREFIX:-}/include:${CONDA_PREFIX:-}/lib ${cufile_search}"
+    fi
+    if [[ ! -z "${PREFIX+x}" ]]
+    then
+      cufile_search="${PREFIX:-}/include:${PREFIX:-}/lib ${cufile_search}"
+    fi
     local gds_version=1.0.0
```

Comment:
Think this is ok on CUDA 12, but may want to double check

Reply:
I see the same file names and locations as checked here in my CUDA 12.2 installation so it should be fine

Reply:
Yeah for context the use case we are testing here is CTK packages installed by Conda from `conda-forge`. The layout may differ relative to other formats (like Conda packages from the `nvidia` channel or the CTK run script)

Reply:
Actually it looks like we only use this to download some GDS version when we can't find one. So think we can ignore this for our use case here